sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt

```diff
@@ -20,21 +19,24 @@ class KittenBinder(
     }
 
     fun onStart() {
-        disposables +=
-            requireNotNull(view)
-                .events
-                .map(KittenViewEventToIntentMapper::invoke)
-                .subscribe(onNext = store::accept)
-
-        disposables +=
-            store
-                .states
-                .map(KittenStateToViewModelMapper::invoke)
-                .subscribe(onNext = { requireNotNull(view).show(it) })
+        startStopScope = disposableScope { start() }
```

Comment:
It would be nice if the example used a non-nullable val for the scope, and then added to it like in the previous impl.

Reply:
How do you imagine this? `lateinit`? `onStart`-`onStop` can be called multiple times before `onDestroy`.

Reply:
I was thinking the same way with CompositeDisposable, with disposables.clear() in onStop.

Reply:
Because it's lifecycle is between start and stop, I think better to keep it nullable and assign null on stop.